### PR TITLE
Add TR/2022/notifications-protocol-20221231

### PR DESCRIPTION
**Pull request by csarven**
Add Solid Notifications Protocol, Version 0.2.0.

`#change-log` section indicates changes since [TR/2022/notifications-protocol-20220502](https://solidproject.org/TR/2022/notifications-protocol-20220509).

Accompanied with https://github.com/solid/vocab/pull/85 which adds the Solid Notifications Protocol vocabulary and JSON-LD context.

---

[Preview](http://htmlpreview.github.io/?https://github.com/solid/specification/blob/5d5da63a7adf8f6fa170fc2f3abd9a6ab4535cba/notifications-protocol.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fspecification%2F3db07b720bae255f98a35a13d96ed41a4995bf1b%2Fnotifications-protocol.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fsolid%2Fspecification%2F5d5da63a7adf8f6fa170fc2f3abd9a6ab4535cba%2Fnotifications-protocol.html)

### CODE REVIEW

File: 2022/notifications-protocol-20221231.html [hunk 1]
```diff
@@ -0,0 +1,1035 @@
...
+          </section>
+
+          <section id="sotd" inlist="" rel="schema:hasPart" resource="#sotd">
+            <h2 property="schema:name">Status of This Document</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This section describes the status of this document at the time of its publication.</p>
+
+              <p>This document was published by the <a href="https://www.w3.org/community/solid/">Solid Community Group</a> as <em>Version 0.2.0</em>. The sections that have been incorporated have been reviewed following the <a href="https://github.com/solid/process">Solid process</a>. However, the information in this document is still subject to change. You are invited to <a href="https://github.com/solid/notifications/issues">contribute</a> any feedback, comments, or questions you might have.</p>
+
+              <p>Publication as <em>Version 0.2.0</em> does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr> Membership. This document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.</p>
```
Comment (TallTed) [inline 2022/notifications-protocol-20221231.html:293]:
```suggestion
              <p>Publication as <em>Version 0.2.0</em> does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr> Membership. This document may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.</p>
```

File: 2022/notifications-protocol-20221231.html [hunk 2]
```diff
@@ -0,0 +1,1035 @@
...
+
+                  <p><a href="#ResourceServer">Resource Servers</a> enable <a href="#SubscriptionClient">Subscription Clients</a> to discover <a href="#description-resource">description resources</a> that are associated with a topic of interest.</p>
+
+                  <p about="#description-resource" property="skos:definition" typeof="skos:Concept">A <strong><dfn id="description-resource" property="skos:prefLabel">description resource</dfn></strong> is an <em>RDF document</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] that includes information about the capabilities and features of a <cite><a href="#subscription-resource-data-model" rel="rdfs:seeAlso">subscription resource</a></cite> or <cite><a href="#notification-channel-data-model" rel="rdfs:seeAlso">notification channel</a></cite>.</p>
+
+                  <p>Advertising <a href="#subscription-resource">subscription resources</a> can be used to create notification channels:</p>
+
+                  <ul>
+                    <li>for protected resources, which relies on capability URL to protect the channel;</li>
```
Comment (TallTed) [inline 2022/notifications-protocol-20221231.html:529]:
```suggestion
                    <li>for protected resource, which relies on capability URL to protect the channel;</li>
```

File: 2022/notifications-protocol-20221231.html [hunk 3]
```diff
@@ -0,0 +1,1035 @@
...
+
+                      <div class="note" id="notification-channel-types-index" inlist="" rel="schema:hasPart" resource="#notification-channel-types-index">
+                        <h5 property="schema:name"><span>Note</span>: Notification Channel Types Index</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Notification channel types are defined and published independently of this document. To help with the discovery of notification channel types that can be used with this specification, it is encouraged to register them by following the procedure given in <cite><a href="https://solidproject.org/TR/notification-channel-types">Notification Channel Types</a></cite> [<cite><a class="bibref" href="bib-notification-channel-types">NOTIFICATION-CHANNEL-TYPES</a></cite>].</p>
+
+                          <p>When defining IRIs for use with a notification channel type, it is considered best practice to publish a vocabulary and JSON-LD context documents that are publicly accessible.</p>
```
Comment (TallTed) [inline 2022/notifications-protocol-20221231.html:631]:
```suggestion
                          <p>When defining IRIs for use with a notification channel type, it is considered best practice to publish vocabulary and JSON-LD context documents that are publicly accessible.</p>
```

**Comment by csarven** [comment]
@TallTed , sorry, I can't accept editorial changes on this document in this repository. Duplicates work, if not potentially introduces divergence that needs to be resolved. If you'd like to process - and I would very much **LOVE** you to - the ED of Solid Notifications Protocol, please have a go at https://github.com/solid/notifications . What makes it into this repository is fundamentally about going through some process and meeting some guidelines.

**Comment by csarven** [comment]
Merging as per consensus in 2023-03-08 meeting: https://github.com/solid/specification/blob/main/meetings/2023-03-08.md#add-tr2022notifications-protocol-20221231

**Pull request merged**